editor/components/default-block-appender/index.js

```diff
@@ -45,10 +44,10 @@ export default compose(
 		( state, ownProps ) => {
 			const isEmpty = ! getBlockCount( state, ownProps.rootUID );
 			const lastBlock = getBlock( state, ownProps.lastBlockUID );
-			const isLastBlockDefault = get( lastBlock, 'name' ) === getDefaultBlockName();
```

Comment:
This was not possible originally because we didn't have the `isUnmodifiedDefaultBlock` function